braininventory/get.py

```diff
@@ -100,3 +100,5 @@ def report():
 
 	return report
 
+def __get_project(df): 
+    return df['project'].value_counts().to_dict() #returns an int of the number of unique projects in the dataframe
```

Comment:
What I am looking for is something along the lines of 

```
def __get_number_of_projects(df): 
    '''
    Returns the number of unique projects in dataframe.
    '''
    return df['project'].value_counts().to_dict()

Reply:
@kexin-chen4 very clever that you converted it to a dictionary. That's certainly the right answer.

![](https://media2.giphy.com/media/j6wQILdEcwUqdy6MJH/giphy.gif?cid=ecf05e47koe99v14lacztkbmu6cumkexdkv9ajkgiqwdgcmr&ep=v1_gifs_search&rid=giphy.gif&ct=g)